lib/net/pktdrv/pktdrv.c

```diff
@@ -42,6 +42,9 @@
 extern unsigned long times(void *);
 
 //temporary dirty interface
+void* PktdrvMmioBase = (void*)0xFEF00000;

```

Comment:
Let's make this a `const` so the compiler can avoid doing reads to get the base address.